src/serialize.ts

```diff
@@ -124,7 +126,8 @@ export default function serialize(
     children = BREAK_TAG;
   }
 
-  if (children === '') return;
+  if (children === '' && !VOID_ELEMENTS.find((k) => nodeTypes[k] === type))
+    return;
```

Comment:
drawing attention here. Let me know if you have comments on the `VOID_ELEMENTS` approach

Reply:
Seems fine to me